src/converters/people_converter.py

```
@@ -0,0 +1,149 @@
+                    "SCHEDULE:YEAR",
+                    "SCHEDULE:CONSTANT",
+                    "SCHEDULE:FILE",
+                ]:
+                    if self.idf.getobject(sched_type, sched_name):
+                        exists = True
+                        break
+                if not exists:
+                    raise ValueError(
+                        f"{field_desc} '{sched_name}' referenced in People '{val_data.name}' "
+                        f"does not exist in IDF."
+                    )
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)